ffi/capi/Cargo.toml

```diff
@@ -104,6 +104,12 @@ compiled_data = [
     "icu_timezone?/compiled_data",
 ]
 
+std = []
+# Adds a looping panic handler (useful for no_std targets)
+panic-handler = []
+# Uses malloc as the global allocator (useful for no_std targets)
+libc-alloc = ["dep:libc_alloc"]
```

Comment:
Nits on feature names:

1. It seems we're using `_` for feature names so we should be consistent
2. `panic-handler` should be more specific, like `looping_panic_handler`